Modules/Core/Common/include/itkVectorContainer.h

```diff
@@ -561,6 +561,18 @@ class ITK_TEMPLATE_EXPORT VectorContainer
     , VectorType(first, last)
   {}
 };
+
+
+/** Makes a VectorContainer that has a copy of the specified `std::vector`. */
+template <typename TElement>
+auto
+MakeVectorContainer(std::vector<TElement> stdVector)
+{
+  auto vectorContainer = VectorContainer<SizeValueType, TElement>::New();
```

Comment:
Thanks for your approvals, @dzenanz and @thewtex Please note that it proposes for `itk::MakeVectorContainer` to return a VectorContainer whose `TElementIdentifier` template argument is `itk::SizeValueType`. I think that's usually what users would expect.  The commonly used `itk::IdentifierType` is also equal to `itk::SizeValueType`, by definition: https://github.com/InsightSoftwareConsortium/ITK/blob/0b03a81a6cc1d7ec0adfdc9d6045cca357115dfe/Modules/Core/Common/include/itkIntTypes.h#L73 Moreover, `itk::SizeValueType` is typically equivalent to `std::vector::size_type` as well.

So yes, I think that this PR is ready as well.  😃